### DISCUSSION TITLE
Support for process Level metrics

### DISCUSSION THREAD

**Discussion by fooris** [open-telemetry/community#1194] [Community Discussions]
Hi all,

we are currently already using open-telemetry in our project to gather application level metrics.

But we would also like to get process-level metrics such as memory usage, thread count, etc.
Does open-telemetry already support this? I seem to be unable to find anything in that regard.

If this is not currently supported, I would be interested in implementing this.

Cheers,
Floris

**Comment by tigrannajaryan** [comment]
Process metrics specification is here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md

They can be collected using Otel Collector's hostmetricsreceiver/process scraper: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver

**Reply by fooris** [reply]
Thats what I was looking for, thank you!
